packages/release-notes-generator/src/index.ts

```diff
@@ -32,14 +28,16 @@ async function run() {
 	const markdown = generateMarkdown(notices, types, untypedPackages, packageVersions);
 
 	const divider = '==============================================================';
-	process.stdout.write(`${divider}\n${markdown}\n${divider}\n`);
+	process.stdout.write(`${divider}\nDirectus v${mainVersion}\n${divider}\n${markdown}\n${divider}\n`);
 
 	const githubOutput = process.env['GITHUB_OUTPUT'];
 
 	// Set output if running inside a GitHub workflow
 	if (githubOutput) {
 		const outputs = [
-			`DIRECTUS_MAIN_VERSION=${mainVersion}`,
+			`DIRECTUS_VERSION=${mainVersion}`,
+			`DIRECTUS_PRERELEASE=${isPrerelease}`,
+			...(prereleaseId ? `DIRECTUS_PRERELEASE_ID=${prereleaseId}` : []),
```

Comment:
```suggestion
			...(prereleaseId ? [`DIRECTUS_PRERELEASE_ID=${prereleaseId}`] : []),
```

I believe this will be unintentionally spread into `'D','I','R'...` without the inclusion of array bracket, if I'm not mistaken

Reply:
Great finding! 🙏